Assets/MirageSDK/Examples/UseCases/Index/UseCaseIndex.cs

```diff
@@ -17,8 +16,8 @@ public class UseCaseIndex : MonoBehaviour
 
         void Start()
         {
-            if(WalletConnect.IsSessionSaved()) 
-                EnableUseCaseMenu();
+           /* if(WalletConnect.IsSessionSaved()) 
```

Comment:
do not leave unecessary comments. if you don't need this script - remove it from scene